common/pkcs11.h

```diff
@@ -371,16 +371,33 @@ typedef unsigned long ck_key_type_t;
 #define CKK_AES			(0x1fUL)
 #define CKK_BLOWFISH		(0x20UL)
 #define CKK_TWOFISH		(0x21UL)
+#define CKK_SECURID		(0x22UL)
+#define CKK_HOTP		(0x23UL)
+#define CKK_ACTI		(0x24UL)
+#define CKK_CAMELLIA		(0x25UL)
+#define CKK_ARIA		(0x26UL)
+#define CKK_MD5_HMAC		(0x27UL)
+#define CKK_SHA_1_HMAC		(0x28UL)
+#define CKK_RIPEMD128_HMAC	(0x29UL)
+#define CKK_RIPEMD160_HMAC	(0x2aUL)
+#define CKK_SHA256_HMAC		(0x2bUL)
+#define CKK_SHA384_HMAC		(0x2cUL)
+#define CKK_SHA512_HMAC		(0x2dUL)
+#define CKK_SHA224_HMAC		(0x2eUL)
+#define CKK_SEED		(0x2fUL)
+#define CKK_GOSTR3410		(0x30UL)
+#define CKK_GOSTR3411		(0x31UL)
+#define CKK_GOST28147		(0x32UL)
 #define CKK_VENDOR_DEFINED	((unsigned long) (1UL << 31))
-
+#define CKK_EC_EDWARDS		(CKK_VENDOR_DEFINED|0x8003UL)
```

Comment:
This seems to be part of PKCS#11 3.0.

Reply:
https://github.com/oasis-tcs/pkcs11/blob/master/working/3-00-wd-01/pkcs11t.h

Reply:
CKK_EC_EDWARDS was added to SoftHSM before PKCS#11 3.0. That is why it was a vendor defined value. Nothing that should be spread to other implementations. SoftHSM should be updated with the values from 3.0.

Reply:
Also according to the chair of the working group these are the final values, and are not expected to change.

Reply:
Great, thanks for verifying, @nmav. I'll move vendor-specific values for EC EDWARDS and EDDSA to use 3.0 version and resubmit.